spec/requests/admin/broadcasts_spec.rb

```diff
@@ -69,7 +69,6 @@
         expect do
           delete admin_broadcast_path(broadcast.id)
         end.to change { Broadcast.all.count }.by(-1)
-        expect(response.body).to redirect_to admin_broadcasts_path
```

Comment:
Removed these `redirect_to` expectations, since the `destroy` action is now async.